raiden/ui/cli.py

```diff
@@ -402,14 +401,16 @@ def options(func):
             "Mediation Fee Options",
             option(
                 "--flat-fee",
-                help=("Flat fee required for every mediation in wei of the mediated token."),
-                default=DEFAULT_MEDIATION_FLAT_FEE,
-                type=FeeAmount,
-                show_default=True,
+                help=(
```

Comment:
Do we have CLI tests that check that args are read correctly and passed on to the config we want?

If yes would be nice to have one for this argument, just to be sure it does what we think it does.

Reply:
Not that I'm aware of, but we should add these.

In this case it's quite nice that we can use `type=(ADDRESS_TYPE, click.IntRange(min=0)),`. This makes sure that there are exactly 2 arguments, and both of them get validated by the defined types.